src/WorkflowManager/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -802,6 +802,7 @@ private async Task<WorkflowInstance> CreateWorkflowInstanceAsync(WorkflowRequest
                 var firstTask = workflow.Workflow.Tasks.First();
 
                 var task = await CreateTaskExecutionAsync(firstTask, workflowInstance, message.Bucket, message.PayloadId.ToString());
+                task.TaskPluginArguments["workflow_name"] = workflow.Workflow.Name;
```

Comment:
Just wondering if we need to remove the validation for workflow name in the validator if we don't need the developer to input it. I think it's a requirement at the minute.

Reply:
@lillie-dae I've just checked and looks like we use the name property and don't look for it in task plugin args so ignore this 🙂